### ISSUE TITLE
DiaNN as a conda package

### ISSUE THREAD

**Issue by bgruening** [vdemichev/DiaNN#334]
A conda package would be very nice for this tool. This would enable easy integration in workflow systems and also create a BioContainer as well. Are there any plans in this direction?

**Comment by ypriverol** [comment]
@bgruening I have spoked with @vdemichev and he is willing to help on this issue. @vdemichev Can you let us know the dependencies of DIA-NN to be able to create a package.

**Comment by ypriverol** [comment]
@vdemichev can we have in the releases as part of the files the .exe that can be use in Linux, not the installation Setup.  conda do not allow me to install based on `rpm` or `deb`. What do you think @bgruening

**Comment by vdemichev** [comment]
Hi Yasset, the installer on Linux just unpacks files and does not do anything else. So the same files can be packaged into a conda package? The dependency, as far as I know, is glibc 2.27.

**Comment by ypriverol** [comment]
You mean if in Linux we do `./DIA-NN.1_8.Setup.exe` will only un-compress the exe in the path?  https://github.com/vdemichev/DiaNN/releases/download/1.8/DIA-NN.1_8.Setup.exe

**Comment by vdemichev** [comment]
No, can unpack .deb and then do sth with the binary and libs?

**Comment by ypriverol** [comment]
Automatically is not possible with bioconda @bgruening. ?

**Comment by bgruening** [comment]
We prefer to compile it on our own with our toolchain. This is will also avoid the glib error. We ensure that the program works then also on HPC for example.

@vdemichev is there a way to compile it on our own?

**Comment by vdemichev** [comment]
No, unfortunately not, can only provide binaries. Glibc dependency is there because need C++17 support.

**Comment by ypriverol** [comment]
We have manage to produce a `docker` and `singularity` containers. As @bgruening mentioned if the tool cannot be compiled by bioconda team will be only available in a few architectures. With the docker and singularity containers we are "save" for now. Thanks a lot to both.

**Comment by bgruening** [comment]
> Automatically is not possible with bioconda @bgruening. ?

This is possible.

> No, unfortunately not, can only provide binaries. Glibc dependency is there because need C++17 support.

Please correct me, but C++17 support is dependent of the compiler not glib, isn't it? If you compile the software on an older box, e.g. CentOS 7 it would work I think.
